docusaurus.config.js

```diff
@@ -93,6 +93,10 @@ module.exports = {
                 label: 'Stacks.js',
                 href: 'https://stacks.js.org',
               },
+              {
+                label: 'Stacks Connect',
+                href: 'https://connect.stacks.js.org/modules/_stacks_connect',
```

Comment:
Perhaps better to point to https://connect.stacks.js.org; otherwise, looks good!
```suggestion
                href: 'https://connect.stacks.js.org',
```

Reply:
👍